package.json

```diff
@@ -15,6 +15,7 @@
     "service-worker"
   ],
   "dependencies": {
+    "chalk": "^4.1.0",
```

Comment:
I assume this is already in `next` dependency chain, so maybe not needed to add here?